ansible-ipi-install/roles/installer/tasks/08_cleanup_sub_man_registeration.yml

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
should we only unregistre if registered?

Reply:
You wouldn't be able to get this far in the playbook if you were not registered (unless you happen to get all those dependencies some other way). Also since it is using a module, shouldn't matter.